open_academy/demo/demo.xml

```diff
@@ -1,16 +1,19 @@
+<?xml version="1.0"?>
 <odoo>
-    <data>
-      <record model="open_academy.course" id="course0">
-        <field name="name">Javascript Course</field>
-        <field name="description">Javascript Course</field>
-      </record>
-      <record model="open_academy.course" id="course0">
-        <field name="name">React Course</field>
-        <field name="description">React Course</field>
-      </record>
-      <record model="open_academy.course" id="course0">
-        <field name="name">Python Course</field>
-        <field name="description">Python Course</field>
-      </record>
-    </data>
+  <data>
+    <record id="course0" model="open_academy.course">
```

Comment:
please change the id names for something with a better description like course_javascript_demo